zipline/data/us_equity_pricing.py

```diff
@@ -204,9 +189,34 @@ class BcolzDailyBarWriter(object):
         'volume': float64,
     }
 
-    def __init__(self, filename, sessions, calendar):
+    def __init__(self, filename, calendar, start_session, end_session):
+        """
```

Comment:
Including the parameters to `__init__`?  Seems strange to have them separate from the method that uses them.